content/recipes/prisma.md

```diff
@@ -88,6 +88,8 @@ Now, open up `.env` and adjust the `DATABASE_URL` environment variable to look a
 DATABASE_URL="file:./dev.db"
 ```
 
+Make sure [you have a ConfigModule configured](https://docs.nestjs.com/techniques/configuration), otherwise the `DATABASE_URL` variable will not be picked up from `.env`.
```

Comment:
```suggestion
Make sure you have a [ConfigModule](https://docs.nestjs.com/techniques/configuration) configured, otherwise the `DATABASE_URL` variable will not be picked up from `.env`.
```